src/App.jsx

```diff
@@ -1,13 +1,16 @@
-import './App.css'
+import "./App.css";
+import { Button } from "./components/Button.component";
+import Footer from "./components/Footer.component";
 
 function App() {
-
-
   return (
-    <main className='flex justify-center items-center w-screen h-screen bg-background'>
-
-    </main>
-  )
+    <>
+      <main className="flex justify-center items-center w-screen h-screen bg-background">
+        <h1>welcome to cocktails recipe app</h1>
+      </main>
+      <Footer />
```

Comment:
I see no problem with width. There is problem with alignment because you add `<Footer/>` outside of `<main>` element